manifests/0000_90_cluster-authentication-operator_02_servicemonitor.yaml

```diff
@@ -53,3 +53,48 @@ spec:
   selector:
     matchLabels:
       app: oauth-openshift
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: openshift-oauth-apiserver
+  namespace: openshift-oauth-apiserver
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    metricRelabelings:
+    - action: drop
+      regex: etcd_(debugging|disk|request|server).*
```

Comment:
we can remove the `request` from the drop list. Similar to kube-apiserver `etcd_(debugging|disk|server).*`
https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml#L52

I realize that I didn't fix it for openshift-apiserver. https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/manifests/0000_90_openshift-apiserver-operator_04_servicemonitor-apiserver.yaml#L49. 

This is basically a histogram that shows the apiserver -> etcd latencies, comes handy in troubleshooting.